test/index.test.js

```diff
@@ -499,6 +499,42 @@ describe('Optimizely', function() {
       });
     });
 
+    describe('#options.sendRevenueOnlyForOrderCompleted', function() {
+      beforeEach(function() {
+        analytics.stub(window.optimizely, 'push');
+      });
+
+      it('should send revenue only on Order Completed if onlySendRevenueOnOrderCompleted is enabled', function() {
```

Comment:
cool -- i'd probably add in this test that sending a non Order Completed event does not include revenue.